SmartDeviceLink/SDLUSBMUXDTransport.m

```diff
@@ -0,0 +1,167 @@
+//  SDLUSBMUXDTransport.m
```

Comment:
@conner-wu Please add copyright for this new file, if it's copyed from from other open source project, just keep the original copyright, otherwise, please add SDLC copyright, you can reference [smartdevicelink](https://github.com/smartdevicelink)